doc/introduction/measurements.rst

```diff
@@ -153,6 +154,47 @@ so corresponds to a :math:`99.75\%` probability of measuring
 state :math:`|00\rangle`, and a :math:`0.25\%` probability of
 measuring state :math:`|01\rangle`.
 
+State
+-----
+
+Support for returning the quantum state of the QNode is also provided. Similar to the
+:func:`~.pennylane.probs` measurement function, **observables should not be input** into the
+:func:`~.state` function. Moreover, the state must be returned over **all** wires in the device
+so that it is pure. For example:
+
+.. code-block:: python3
+
+    from pennylane.beta.queuing import state
+    from pennylane.beta.tapes import QNode
```

Comment:
I'm a bit worried about how many things are now dependent on code from the beta folder. Is it unavoidable?

Reply:
I think that is a great idea: the state feature would be silent for now, and when all the folders are tidy we advertise it in the docs?

Reply:
I will remove for now, at least we know where to look when we want to add in again :grin: